src/EnergyPlus/MixedAir.cc

```diff
@@ -672,11 +686,19 @@ namespace MixedAir {
 			// Coil Types
 		} else if ( SELECT_CASE_var == WaterCoil_Cooling ) { // 'Coil:Cooling:Water'
 			if ( Sim ) {
```

Comment:
switched from ControlCompOutput to RegulaFalsi for water coils in the main air loop